terraform/aks/databases.tf

```diff
@@ -0,0 +1,36 @@
+locals {
+  postgres_connection_string = "Server=${module.postgres.host};Database=${module.postgres.name};Port=${module.postgres.port};User Id=${module.postgres.username};Password='${module.postgres.password}';Ssl Mode=Require;Trust Server Certificate=true"
```

Comment:
Can we use the new module output?

Reply:
The `url` output has a similar problem to Redis in that it uses a format that doesn't work for .NET. I could add a `connection_string` output to the postgres module similar to what we did for Redis?

Reply:
Yes let's do it now